services/news/src/views/multiple-ad.ejs

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport"
+        content="width=device-width, initial-scale=1.0" />
+  <title><%= TITLE %></title>
+  <link rel="stylesheet"
+        href="/css/global.css" />
+  <link rel="stylesheet"
+        href="/css/unified.css" />
+  <link rel="icon"
+        href="/img/spy.svg" />
+  <script src="/js/main.js"></script>
+  <script>
+    // Seller acting as ad server is included implictly, and their tag is
+    // included on the page directly. This configuration field should list
+    // additional sellers to include in the ad auctions.
+    const otherSellers = window.PSDemo.getQueryAsArray('otherSellers') || [
+      '<%= SSP_ORIGIN %>',
+      '<%= SSP_A_ORIGIN %>',
+      '<%= SSP_B_ORIGIN %>',
```

Comment:
Q: To add more sellers to the auction (e.g. SSP-X), would their origin have to be added here? How does the window function with otherSellers work?

Reply:
Renamed to getUrlQueryAsString and getUrlQueryAsArray, both of which are utility functions from https://news/js/main.js.

This snippet of code looks for a URL query called "otherSellers", parses it as an array of comma separated values if found or just defaults to using this list.

Easiest way to include more sellers would be to just add to the list as you mentioned. And a new demo page on the news site could have its own set of default "otherSellers".